README.md

```diff
@@ -12,3 +12,27 @@ This is the home for the Gorila team's components, built by developers for devel
 See our [Getting Started Guide](./additional-documentation/getting-started.html) if you're building a project with ours widgets.
 
 If you'd like to contribute, you must follow our [contributing guidelines](./additional-documentation/contributing.html).
+
+### Developing
```

Comment:
These documentation needs to be in "Development Documentation" in http://ui.gorilainvest.com.br/docs, can you move it please?

Reply:
Also, the refered steps must be done by ng/nx cli, and not manually

Reply:
I think that is very wired the readme file doesn't contains the information about how to use a repository.

Reply:
Okay, but this is out of scope for this pull request, and wasn't made with a specific commit, with this I mean that this file was mixed with another unrelated files in the same commit.

Reply:
if this were the pattern within this repository, contribution guidelines and get started should already be in the readme, agree?

Reply:
> Also, the refered steps must be done by ng/nx cli, and not manually

Where's the instructions? I can't find this steps in documentation. Also, I can't find documentation of existent components.

Reply:
http://ui.gorilainvest.com.br/docs/

Reply:
In this link you didn't explain the two indirect questions above:
1) where is the information about how to use ng/nx cli to create a new component without do steps manually?
2) how to add information in the docs?